libkode/printer.cpp

```diff
@@ -631,6 +632,11 @@ QString Printer::functionSignature( const Function &function,
   return s;
 }
 
+void Printer::setVerbose(bool verbose)
```

Comment:
Space within parenthesis, in this code base. (as much as I don't like it, it should at least be consistent)

(to be done everywhere)